Shared/mods/deathmatch/logic/luadefs/CLuaUtilDefs.cpp

```diff
@@ -9,6 +9,47 @@
  *****************************************************************************/
 
 #include "StdInc.h"
+#ifdef MTA_CLIENT
+#include <SharedUtil.SysInfo.h>
+#include <SharedUtil.SysInfo.hpp>
+#endif
+#include <lua/CLuaFunctionParser.h>
+
+#ifdef MTA_CLIENT
+namespace mta
+{
+const auto& GetSystemInfo()
+{
+    using DataSubTable = std::unordered_map<std::string, std::variant<std::string, lua_Number>>;
+    using MainTable = std::unordered_map<std::string, std::variant<DataSubTable, lua_Number>>;
+
+    static MainTable outValue{};
+    if (outValue.empty()) {
+        {
+            auto info = SharedUtil::GetWMISystemInfo();
+            outValue.emplace("CPU", DataSubTable{
+                {"MaxClockSpeed", (lua_Number)info.CPU.MaxClockSpeed},
+                {"Name", info.CPU.Name},
+                {"NaNumberOfCores", (lua_Number)info.CPU.NumberOfCores},
+                {"NumberOfLogicalProcessors", (lua_Number)info.CPU.NumberOfLogicalProcessors},
+            });
+            outValue["TotalPhysicalMemory"] = (lua_Number)info.TotalPhysicalMemory;
+        }
+
+        {
+            BasicGPUInfo gpu;
+            g_pCore->GetGraphics()->GetRenderItemManager()->GetBasicGPUInfo(gpu);
+            outValue.emplace("GPU", DataSubTable{
+                { "RAM", (lua_Number)gpu.memoryKB},
+                { "Name", gpu.name },
```

Comment:
Inconsistent use of whitespace.
```suggestion
                {"RAM", (lua_Number)gpu.memoryKB},
                {"Name", gpu.name},
```